storage/range.go

```diff
@@ -1593,6 +1643,12 @@ func (r *Range) InternalLeaderLease(batch engine.Engine, ms *proto.MVCCStats, ar
 			return r.ContainsKey(configPrefix)
 		})
 		r.tsCache.SetLowWater(oldLease.Expiration.Add(int64(r.rm.Clock().MaxOffset()), 0))
+
+		if log.V(1) {
+			nodeID, storeID := DecodeRaftNodeID(multiraft.NodeID(args.Lease.RaftNodeID))
```

Comment:
Actually, let's make this log statement info, not verbose(1). This will be helpful.